bip-0155.mediawiki

```diff
@@ -120,21 +120,23 @@ The list of reserved network IDs is as follows:
 | Cjdns overlay network address
 |}
 
-To allow for future extensibility, clients MUST ignore address types that they do not know about.
-Client MAY store and gossip address formats that they do not know about. Further network ID numbers MUST be reserved in a new BIP document.
+Clients are RECOMMENDED to gossip addresses from all known networks even if they are currently not connected to some of them. That could help multi-homed nodes and make it more difficult for an observer to tell which networks a node is connected to.
 
-Clients SHOULD reject addresses that have a different length than specified in this table for a specific address ID, as these are meaningless.
+Clients SHOULD NOT gossip addresses from unknown networks because they have no means to validate those addresses and so can be tricked to gossip invalid addresses.
+
+Further network ID numbers MUST be reserved in a new BIP document.
+
+Clients SHOULD reject messages that contain addresses that have a different length than specified in this table for a specific network ID, as these are meaningless.
 
 See the appendices for the address encodings to be used for the various networks.
 
-==Compatibility==
+==Signaling support and compatibility==
+
+Introduce a new message type <code>sendaddrv2</code>. Sending such a message indicates that a node can understand and prefers to receive <code>addrv2</code> messages instead of <code>addr</code> messages. I.e. "Send me addrv2".
```

Comment:
s/. I.e./, e.g./

Reply:
> If it's kept, then I think "i.e." is correct.  It's a rewording of the previous sentence, not giving an example of a larger class of things.

I agree.

Still prefer s/. I.e./, i.e./

Reply:
Yep, agree i.e. is preferred over I.e.

I guess ideally (in the case of it being kept) it would be something like:
... messages, i.e. "Send me addrv2".
... messages - i.e. "Send me addrv2".